nilearn/datasets/struct.py

```diff
@@ -118,6 +120,75 @@ def load_mni152_template():
     return check_niimg(path)
 
 
+def load_mni152_brain_mask():
+    """Load brain mask from MNI152 template
+
+    Returns
+    -------
+    mask_img: Nifti-like mask image corresponding to grey and white matter.
+
+    References
+    ----------
+    Refer to load_mni152_template function for more information about MNI
+    template
+    """
+    # Load MNI template
+    target_img = load_mni152_template()
+    mask_voxels = (target_img.get_data() > 0).astype(int)
+    mask_img = new_img_like(target_img, mask_voxels)
+    return mask_img
+
+
+def fetch_mni152_grey_matter_mask(data_dir=None, resume=True, verbose=1):
+    """Downloads ICBM152 template first, then loads computed 'gm' mask image.
+
+    The output 'gm' mask image will be resampled to MNI152 template space.
+
+    Parameters
+    ----------
+    data_dir: str, optional
+        Path of the data directory. Used to force storage in a specified
+        location. Defaults to None.
+
+    resume: bool, optional
+        If True, try resuming partially downloaded data. Defaults to True.
+
+    verbose: int, optional
+        verbosity level (0 means no message).
+
+    Returns
+    -------
+    gm_mask_img: Nifti image
+        Corresponding to brain grey matter resampled to MNI152 template space.
+
+    See Also
+    --------
+    nilearn.datasets.fetch_icbm152_2009: for details regarding the ICBM152
+        template.
+
+    nilearn.datasets.load_mni152_template: for details regarding MNI152 template.
+
+    nilearn.image.resample_to_img: to resample source image to target image.
+    """
```

Comment:
> The mask seems very permissive, why .2? .5 seems more logical to me.

Let's make it an optional argument. Permissive masks are usual a good
idea, however, so I think that .2 is a good default.